microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+	* [CoreWebView2SharedBuffer:: Close method](#)
+	* [CoreWebView2SharedBuffer:: get_Size method](#)
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_17 interface](#)
+	* [ICoreWebView2_17::PostSharedBufferToScript method](#)
+* [ICoreWebView2Environment12 interface](#)
+	* [ICoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [ICoreWebView2Frame4 interface](#)
+	* [ICoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [ICoreWebView2SharedBuffer interface](#)
+	* [ICoreWebView2SharedBuffer:: OpenStream method](#)
+	* [ICoreWebView2SharedBuffer:: Close method](#)
+	* [ICoreWebView2SharedBuffer:: get_Size method](#)
+	* [ICoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [ICoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+---
+
+*  The Permission API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Profile4 interface](#)
+	* [CoreWebView2Profile4::SetPermissionState method](#)
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [CoreWebView2PermissionSetting interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Profile4 interface](#)
+	* [CoreWebView2Profile4::SetPermissionState method](#)
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [CoreWebView2PermissionSetting interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)

```

Comment:
```suggestion
* [CoreWebView2ControllerOptions Class](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions?view=webview2-winrt-1.0.1619-prerelease)
   * [CoreWebView2ControllerOptions.LocaleRegion Property](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions?view=webview2-winrt-1.0.1619-prerelease#localeregion)<!--todo: is this the correct property for WinRT, instead of ScriptLocale?-->
* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2getnondefaultpermissionsettingscompletedhandler?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)<!--todo: omit for WinRT?-->
* [CoreWebView2PermissionSetting Class](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease)
   * [CoreWebView2PermissionSetting.PermissionKind Property](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease#permissionkind)
   * [CoreWebView2PermissionSetting.PermissionOrigin Property](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease#permissionorigin)
   * [CoreWebView2PermissionSetting.PermissionState Property](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease#permissionstate)
* [CoreWebView2Profile Class](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1619-prerelease)
   * [CoreWebView2Profile.SetPermissionStateAsync Method](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1619-prerelease#setpermissionstateasync)
   * [CoreWebView2Profile.GetNonDefaultPermissionSettingsAsync Method](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1619-prerelease#getnondefaultpermissionsettingsasync)
* [CoreWebView2SetPermissionStateCompletedHandler Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2setpermissionstatecompletedhandler?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)<!--todo: omit for WinRT?-->
```
tech review: find "todo":  3 questions:
```
   * [CoreWebView2ControllerOptions.LocaleRegion Property](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions?view=webview2-winrt-1.0.1619-prerelease#localeregion)
<!--todo: is this the correct property for WinRT, instead of ScriptLocale?-->

* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2getnondefaultpermissionsettingscompletedhandler?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
<!--todo: omit for WinRT?-->

* [CoreWebView2SetPermissionStateCompletedHandler Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2setpermissionstatecompletedhandler?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
<!--todo: omit for WinRT?-->
```

Reply:
@peiche-jessica - find "todo", omit these 3 links?

Reply:
I need to re-enter the above comment, to fix the version #

Reply:
re:
```
CoreWebView2ControllerOptions.LocaleRegion Property
<!--todo: is this the correct property for WinRT, instead of ScriptLocale?-->
```
`LocaleRegion` was promoted to stable as `ScriptLocale` in m112. 
It’s still experimental in m111, so it won't be in a m111 release package.

Reply:
As of 1671, the API has been promoted to stable in a prerelease package. We should link to the stable ScriptLocale instead of the deprecated experimental LocaleRegion.